tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -38,9 +38,11 @@ describe('Test Fleet access with RBAC with custom roles', { tags: '@rbac' }, ()
       cy.createRoleTemplate({
         roleType: "Global",
         roleName: "fleetAllVerbsRole",
-        newUserDefault: "Yes",
-        resources: ["fleetworkspaces", "gitrepos", "bundles"],
-        verbs: ["create", "delete", "get", "list", "patch", "update", "watch"],
+        resources: [
```

Comment:
IMO `resources` is not valid name anymore for this array, it should be called `rules` instead. Each entry/object is `rule`.

Reply:
`resourcesWithRules` would be appropriate name.

Reply:
> `resourcesWithRules` or `resourcesWithVerbs` would be appropriate name.

I disagree, the corresponding block in ClusterRole resource is called `rules` in k8s, please see https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-examples

Reply:
Comment address in commit `34f79df`